### PR TITLE
925962 - Update Text Settings content in Image Editor UG document.

### PR DESCRIPTION
**Pull request by sanjayvikramsf4373**
### Description: ###

I have updated the Image Editor document to include details about the newly implemented Text Background feature.

### CODE REVIEW

File: wpf/Image-Editor/Text.md [hunk 1]
```diff
@@ -32,6 +35,10 @@ Upon selecting the Text icon in the toolbar, a sub toolbar will be generated bel
 
 >N Text needs to be selected to apply the customization from the sub toolbar.
 
+### Background
+
+You can customize the text background color by clicking the fill icon, selecting color from the color picker, and applying it as the background.
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:40]: You can customize the text background color by clicking the fill icon, choosing a color from the color picker, and applying it to the background.

File: wpf/Image-Editor/Text.md [hunk 2]
```diff
@@ -61,19 +67,27 @@ You can add text to an image using the AddText method programmatically. This met
 * TextSettings - Customizes the text.
 
 {% tabs %} 
+{% highlight XAML %} 
 
-{% highlight C# %} 
-
-editor.AddText("Hello", new TextSettings());
+<editor:SfImageEditor x:Name="editor"
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:72]: @sanjayvikramsf4373 , close editor tag

</editor:SfImageEditor>

File: wpf/Image-Editor/Text.md [hunk 3]
```diff
@@ -61,19 +67,27 @@ You can add text to an image using the AddText method programmatically. This met
...
+}
 
+{% endhighlight %}
 {% endtabs %} 
 
 ## Text settings
 
 Using the following properties in TextSettings, text can be customized.
 
+* [`Background`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.ImageEditor.TextSettings.html#Syncfusion_UI_Xaml_ImageEditor_TextSettings_Background) - Specifies the background color of the text annotation. The default value is Brush.Transparent.
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:90]: Specifies the background color of the text annotation. The default value is `Brush.Transparent.`

File: wpf/Image-Editor/Text.md [hunk 4]
```diff
@@ -86,29 +100,39 @@ Using the following properties in TextSettings, text can be customized.
...
-            textSettings.FontFamily = new FontFamily("Bell MT");
-            textSettings.FontSize = 22;
-            textSettings.Color = new SolidColorBrush(Colors.DarkGreen);
-            textSettings.TextEffects =  TextEffects.Italic;
-            textSettings.Bounds = new Rect(50, 23, 30, 15);
-            textSettings.TextAlignment = TextAlignment.Center;
-            editor.AddText("The happiness of your \nlife depend upon the \nquality of your thoughts.", textSettings);
-
+<editor:SfImageEditor x:Name="editor"                         
+                      Loaded="OnImageEditorLoaded">
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:106]: Close tag is missing. @sanjayvikramsf4373 , So many times repeating same kind of mistakes. As many times explained before adding codes, must ensuring it in simple sample.

**Review by Karthickmani97 — COMMENTED** [review]
@sanjayvikramsf4373 , address review correction.

File: wpf/Image-Editor/Text.md [hunk 5]
```diff
@@ -61,19 +67,27 @@ You can add text to an image using the AddText method programmatically. This met
...
 
-{% highlight C# %} 
-
-editor.AddText("Hello", new TextSettings());
+<editor:SfImageEditor x:Name="editor"                         
+                      Loaded="OnImageEditorLoaded">
 
 {% endhighlight %}
+{% highlight C# hl_lines="3" %}
+
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:77]: Add `this.editor.Loaded += OnImageEditorLoaded;`

File: wpf/Image-Editor/Text.md [hunk 6]
```diff
@@ -86,29 +100,39 @@ Using the following properties in TextSettings, text can be customized.
...
-            textSettings.TextAlignment = TextAlignment.Center;
-            editor.AddText("The happiness of your \nlife depend upon the \nquality of your thoughts.", textSettings);
-
+<editor:SfImageEditor x:Name="editor"                         
+                      Loaded="OnImageEditorLoaded">
 
 {% endhighlight %}
+{% highlight C# %} 
+       
+this.editor.Loaded += OnImageEditorLoaded;
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:111]: Use prefix this - `this.editor.Loaded += this.OnImageEditorLoaded;`

File: wpf/Image-Editor/Text.md (hunk 2 above)
Comment (sanjayvikramsf4373) [reply wpf/Image-Editor/Text.md:72]: closed the editor tag

File: wpf/Image-Editor/Text.md (hunk 5 above)
Comment (sanjayvikramsf4373) [reply wpf/Image-Editor/Text.md:80]: added the line

File: wpf/Image-Editor/Text.md (hunk 3 above)
Comment (sanjayvikramsf4373) [reply wpf/Image-Editor/Text.md:93]: Modified.

File: wpf/Image-Editor/Text.md (hunk 4 above)
Comment (sanjayvikramsf4373) [reply wpf/Image-Editor/Text.md:109]: close tag added .

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**